packages/vite/src/node/plugins/css.ts

```diff
@@ -818,6 +784,79 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
   }
 }
 
+/**
+ * Create a replacer function that takes code and replaces given pure CSS chunk imports
+ * @param pureCssChunkNames The chunks that only contain pure CSS and should be replaced
+ * @param outputFormat The module output format to decide whether to replace `import` or `require`
+ */
+export function getEmptyChunkReplacer(
+  pureCssChunkNames: string[],
+  outputFormat: ModuleFormat,
+): (code: string) => string {
+  const emptyChunkFiles = pureCssChunkNames
+    .map((file) => path.basename(file))
+    .join('|')
+    .replace(/\./g, '\\.')
+
+  // require and import calls might be chained by minifier using the comma operator
+  // in this case we have to keep one comma
+  // if a next require is chained or add a semicolon to terminate the chain.
```

Comment:
Let's also remove this comment temporarily and add it in the other PR later, since it doesn't apply on this existing code.